src/components/navbar.jsx

```diff
@@ -1,41 +1,57 @@
 import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
+import { ImUser } from 'react-icons/im';
 
 export default function Header() {
   const [data, setData] = useState({
     render: true,
-    class: 'links',
+    class: 'links container',
   });
 
   function toggle() {
     const { render } = data;
     if (render === false) {
       setData(() => ({
         render: true,
-        class: 'links',
+        class: 'links container',
       }));
     } else {
       setData(() => ({
         render: false,
-        class: 'links hide',
+        class: 'links container hide',
       }));
     }
   }
-
   return (
     <header className="navbar">
-      <h1 className="header">Bookstore</h1>
-      <nav className="navdiv">
-        {(data.render) ? <button type="button" className="navbutton" onClick={toggle}>menu</button> : <button type="button" className="navbutton" onClick={toggle}>close</button> }
-        <ul className={`${data.class}`}>
-          <li>
-            <button type="button" className="linkbutton" onClick={toggle}><Link to="/" className="link">Home</Link></button>
-          </li>
-          <li>
-            <button type="button" className="linkbutton" onClick={toggle}><Link to="Categories" className="link">Categories</Link></button>
-          </li>
-        </ul>
-      </nav>
+      <div className="navwrap container">
+        <h1 className="header">Bookstore CMS</h1>
+        <nav className="navdiv container">
+          <ul className={data.class}>
+            <li>
+              <button type="button" className="linkbutton" onClick={toggle}><Link to="/" className="link">Home</Link></button>
+            </li>
+            <li>
+              <button type="button" className="linkbutton" onClick={toggle}><Link to="Categories" className="link">Categories</Link></button>
+            </li>
+          </ul>
+        </nav>
+      </div>
+      {(data.render) ? (
+        <button type="button" className="navbutton" aria-label="Save" onClick={toggle}>
+          <ImUser style={{
+            color: '#0290ff', backgroundColor: '#fff', border: '1px solid #e8e8e8', borderRadius: '50%', fontSize: '1.2rem', padding: '5px',
+          }}
+          />
+        </button>
```

Comment:
- [ ] I would like to recommend you to not use inline css Because by using inline CSS, you mix style logic with your component logic, making your code less modular and harder to maintain. It's better to separate style from content by using external CSS files or inline styling approaches like CSS modules. Please fix it 😄